plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/table/rows/functions.tsx

```diff
@@ -43,9 +46,37 @@ function getStatusBadges( subscription: Subscription ): StatusBadge[] {
 		badges.push( {
 			text: __( 'No subscription', 'woocommerce' ),
 			level: StatusLevel.Error,
-			explanation: __(
-				'To get updates and support for this extension, you need to purchase a new subscription, or else share or transfer a subscription for this extension from another account.',
-				'woocommerce'
+			explanation: createInterpolateElement(
+				__(
+					'To receive updates and support, please <purchase>purchase</purchase> a subscription or use a subscription from another account by <sharing>sharing</sharing> or <transferring>transferring</transferring>.',
+					'woocommerce'
+				),
+				{
+					purchase: (
+						<a
+							href={ subscribeUrl( subscription ) }
+							rel="nofollow noopener noreferrer"
+						>
+							renew
+						</a>
+					),
+					sharing: (
+						<a
+							href={ MARKETPLACE_COLLABORATION_PATH }
+							rel="nofollow noopener noreferrer"
+						>
+							sharing
+						</a>
+					),
+					transferring: (
+						<a
+							href={ MARKETPLACE_COLLABORATION_PATH }
+							rel="nofollow noopener noreferrer"
+						>
+							sharing
+						</a>
+					),
```

Comment:
I'm not sure these links make a lot of sense – if you don't have a subscription, you would want somebody else to share theirs with you, or transfer theirs to you. Maybe it would be more helpful to link to https://woo.com/document/managing-woocommerce-com-subscriptions/#transfer-a-woocommerce-com-subscription and https://woo.com/document/managing-woocommerce-com-subscriptions/#transfer-a-woocommerce-com-subscription.

Reply:
Fixed